db/sql/src/tickets.rs

```diff
@@ -206,6 +206,14 @@ impl HoprDb {
             })
             .await?)
     }
+
+    pub async fn reset_ticket_statistics(&self) -> Result<()> {
+        #[cfg(all(feature = "prometheus", not(test)))]
+        {
+            METRIC_HOPR_TICKETS_INCOMING_STATISTICS.reset();
+        }
```

Comment:
Shouldn't you also reset the values in the DB?

Reply:
This would mean that the pending tickets should be removed, and thus lost ?